.gitignore

```diff
@@ -80,7 +80,7 @@ wp-tests-config.php
 /src/wp-content/themes/twentynineteen/node_modules
 /src/wp-content/themes/twentytwentyone/node_modules
 /src/wp-content/themes/twentytwenty/node_modules
-
+/.idea
```

Comment:
I think that this is unintentional. But, in case it's not, because there are so many different text editors, Core will not contain a list of editor specific directories to ignore. Instead, you should configure these in your global Git ignore configuration.

Reply:
Hi @desrosj, changes have been made as per your suggestion.